core/src/main/java/org/mule/runtime/core/internal/processor/strategy/StreamEmitterProcessingStrategyFactory.java

```diff
@@ -152,6 +159,8 @@ protected void stopSchedulers() {
           }
         }
       }
+
+      return shouldStop;
```

Comment:
shouldn't this be `true` only if `super.stopSchedulers()` returned `true` and `flowDispatchScheduler != null`?